docs/02-app/01-building-your-application/04-caching/index.mdx

```diff
@@ -0,0 +1,426 @@
+
+#### Opting out
+
+To opt out of memoization in `fetch` requests, you can pass an `AbortController` `signal` to the request.
+
+```js
+const { signal } = new AbortController()
+fetch(url, { signal })
+```
+
+Since you manually opt into using the `cache` function, there's no way to opt out.
+
+### Data Cache
+
+Next.js has a built-in Data Cache that stores the result of data requests across user requests and deployments.
+
+By default, data requests that use `fetch` are automatically cached. This is possible because Next.js extends the native `fetch` API to allow each request on the server to set its own persistent caching semantics.
+
+You can use the [`cache`](#fetchoptionscache) and [`next.revalidate`](#fetchoptionsnextrevalidate) options of `fetch` to configure the caching behavior. In the browser, `cache` indicates how a fetch request will interact with the browser's HTTP cache, and with Next.js, the `cache` option also indicates how a server-side fetch request will interact with the framework's Data Cache.
+
+When fetching data on the server, a `fetch` request first checks the Data Cache for a cached response. If a cached response is found, it is returned immediately. If not, the request is made to the data source, and the response is cached.
+
+> **Static vs. Dynamic Data Fetching**:
+>
+> You may see the terms **Static Data Fetching** and **Dynamic Data Fetching** being used. Static Data Fetching refers to cached data requests (default), whereas Dynamic Data Fetching refers to data that is fetched on every request, at runtime.
+
+#### Duration
+
+The Data Cache is persistent across user requests and deployments unless you revalidate or opt-out.
+
+#### Revalidating
+
+Revalidation is the process of purging the Data Cache and re-fetching the latest data. This is useful when your data changes and you want to ensure you show the latest information.
+
+Cached data can be revalidated in two ways:
+
+- **Time-based revalidation**: Automatically revalidate data after a certain amount of time has passed. This is useful for data that changes infrequently and freshness is not as critical. Time-based revalidation can be configured with the [Route Segment Config options](#segment-config-options) or per individual [`fetch` request](#fetchoptionsnextrevalidate).
+- **On-demand revalidation**: Manually revalidate data based on an event (e.g. form submission). On-demand revalidation can use a tag-based or path-based approach to revalidate groups of data at once. This is useful for when you want to ensure the latest data is always shown. On-demand revalidation can be configured with the [`revalidateTag`](#fetchoptionsnexttag-and-revalidatetag) and [`revalidatePath`](#revalidatepath) APIs.
+
+#### Opting out
+
+For individual data fetches, you can opt out of caching by setting the [`cache`](#fetchoptionscache) option to `no-store`. This means data will be fetched dynamically, on each request.
```

Comment:
```suggestion
For individual data fetches, you can opt out of caching by setting the [`cache`](#fetchoptionscache) option to `no-store`. This means data will be fetched whenever the fetch is called.
```

Let's move away from saying `on each request` because the layering of caches it could be the entire page becomes static, e.g. when setting force-static, then it would only get called at build time.
Maybe we also need to explain here that no-store opts the route out of the route cache.